backend/cognito/template.yaml

```
@@ -440,9 +440,12 @@ Resources:
       SmsAuthenticationMessage: '{####} is your GOV.UK One Login security code.'
       MfaConfiguration: OPTIONAL
       EmailConfiguration:
-        EmailSendingAccount: COGNITO_DEFAULT
+        EmailSendingAccount: DEVELOPER
+        SourceArn: !Sub arn:aws:ses:${AWS::Region}:${AWS::AccountId}:identity/${DomainOrEmailIdentityToVerify}
+        From: !Sub 'sign-in@${DomainOrEmailIdentityToVerify}'
```

```suggestion
        From: !Sub 'admin-tool@${DomainOrEmailIdentityToVerify}'
```
This makes more sense IMO